source/OptimaJet.DWKit.StarterApplication/BackendServicesExtension.cs

```diff
@@ -17,6 +17,7 @@
 using OptimaJet.DWKit.StarterApplication.Models;
 using OptimaJet.DWKit.StarterApplication.Repositories;
 using OptimaJet.DWKit.StarterApplication.Services;
+using SIL.AppBuilder.BuildEngineApiClient;
```

Comment:
Please remove this "using" and use the fully qualified name so we don't have "Model.Project"